zipkin-cassandra/src/schema/cassandra-schema-cql3.txt

```diff
@@ -0,0 +1,73 @@
+CREATE KEYSPACE "Zipkin" WITH replication = {'class': 'NetworkTopologyStrategy', 'us-west': '1'}  AND durable_writes = true;
```

Comment:
I think two is plenty, since by default the second copy will already be in a different AZ. If there are no updates, then there's no real need for a consistency tiebreaker which is what 3 gives you over 2.

Also, durable writes = true is the default, so you can skip that as well.